test/tools/auth0/handlers/clients.tests.js

```diff
@@ -98,12 +99,14 @@ describe('#clients handler', () => {
     it('should update client', async () => {
       const auth0 = {
         clients: {
-          create: (data) => {
+          create: function(data){
+            expect(this).to.not.be.undefined
```

Comment:
Much of this PR is just asserting that the `this` keyword is defined in all the class methods.